### PR TITLE
chapter 14: Following Users

### PR DESCRIPTION
**Pull request by puteriffah**

### CODE REVIEW

**Comment by puteriffah** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -17,6 +17,13 @@ class User < ApplicationRecord
   before_create :create_activation_digest
   has_secure_password
   has_many :microposts, dependent: :destroy
+<<<<<<< HEAD
```
Comment (ngocvt-0484) [inline app/models/user.rb]: please fix conflict here

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Comment (ngocvt-0484) [inline db/schema.rb]: fix conflict

**Comment by puteriffah** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**